src/views/OrderView.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div class="container">
+        <h1>Order varibles</h1>
+        <toast-component
+        class="toast-component mt-2"
+        v-if="toast"
+        :type="toast.type"
+        :message="toast.message"
+        @toastCloseBtnClicked="clearToast">
+      </toast-component>
+        <div class="row">
+          <div class="col-md-6">
+            <form-component
+            id="order-form"
```

Comment:
Sometimes parameters are inentend other times they are not. Maybe space them the same.